proposal/README.rst

```diff
@@ -174,8 +174,8 @@ Time Table
     - Job shop scheduling
   * - B5
     - 08:20 - 08:38
-    - TBD
-    - TBD
+    - LittleLaGi
```

Comment:
You may not touch other lines.  You seemed to mistakenly included commits cf4934a769c934f2c410de8546c5c65e1ea37bcf and 001f6d3a3f07ca372c2aae7d2cf77a9138edc9f7.  Please do not include unrelated commits.

Reply:
Ok, I didn't notice it before, and I just modified it!